### DISCUSSION TITLE
Unable to proceed ahead after pressing the continue button for GitHub education developer pack

### DISCUSSION THREAD

**Discussion by Einsteinia11** [community/community#141639] [GitHub Education] [answered]
### Hi everyone,

Hello!
I am currently pursuing MCA from Amity University online previously I pursued my BSC in Shaheed Rajguru College of Applied Sciences for women at University of Delhi. I am not able to proceed ahead in my application it asks for location and select this school but after pressing continue button it is not going ahead. Please help I am not able to submit my application !
![image](https://github.com/user-attachments/assets/cbcfb561-ad2c-4840-bb1c-4e26f9ea2bb1)

**Comment by davevad93** [comment, marked as answer]
Hi @Einsteinia11 ,

It could be a browser cache issue, try one of the following solutions:

1. Clear the cache.
2. Use incognito mode.
3. Use another browser.
4. Submit the application via cell phone.

**Reply by Einsteinia11** [reply]
Thank You! It worked in phone but to take the picture of the proof the back camera option is not working it is only able to capture through selfie camera. The back camera option is not working it is redirecting to front camera only!
